PonyGE2/src/scripts/GE_LR_parser.py

```diff
@@ -343,15 +348,39 @@ def main():
 
     # Iterate over the solution list until the target string is parsed.
     solution = parse_target_string()
+    if params['DYNAMIC_BNF'] is True:
```

Comment:
The piece of code where you go to create the dynamic grammar and parse the original code again with the new dynamically created grammar for that specific code.